pal/os/src/windows/az_pal_os.c

```diff
@@ -1,27 +1,19 @@
 // Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+// Licensed under the MIT license.
+// See LICENSE file in the project root for full license information.
 
+#include <stdbool.h>
```

Comment:
is this really needed?

Reply:
No, removed.

---
In reply to: [342857339](https://github.com/Azure/azure-ulib-c/pull/47#discussion_r342857339) [](ancestors = 342857339)